src/Me/components/FormField/FormField.js

```diff
@@ -3,24 +3,42 @@ import styled from 'styled-components';
 import uniqueId from 'lodash/uniqueId';
 import { formFieldContext } from './formContext';
 
+const InlineHelpText = styled.span`
+  color: #5c5c5c;
```

Comment:
Consider importing this value to a (global?) class.
Looks like you done something similer in `EditProfile.css`

Reply:
Actually we want to convert all the global styles into styledComponents.

Reply:
That fine but I can see some use for global classes / styledComponents

Reply:
You're right. We want to change this.

Reply:
oh right i misread your comment. sry.